omnibridge/main.py

```diff
@@ -1,96 +1,40 @@
 import argparse
-
-from omnibridge.flows.branching_flow import BranchingFlow
-from omnibridge.flows.flow_loader import FlowLoader
-from omnibridge.flows.sequential_flow import SequentialFlow
+import os
 from omnibridge.model_entities.models_io.base_model_io import TextualIO
-from omnibridge.saved_data.json_data_manager import JsonDataManager
-from omnibridge.wrappers.api_key import ApiKey
 from omnibridge.cli.banner import banner
-
-from omnibridge.wrappers.wrapper_instances.dalle_wrapper import DALLEWrapper
-from omnibridge.wrappers.wrapper_instances.gpt_wrapper import GPTWrapper
+from omnibridge.cli.argument_parser.argument_parser import add_create_key_sub_parser, \
+    add_create_model_sub_parser, add_create_flow_sub_parser, add_run_flow_sub_parser, \
+    add_list_wrappers_arguments
+from omnibridge.cli.argument_parser.command_handlers import handle_create_command, \
+    handle_list_wrappers_command, handle_run_flow_command
+from omnibridge.saved_data.json_data_manager import FILE_PATH
 from omnibridge.wrappers.wrapper_instances.type_name_to_wrapper import ModelLoader
 
+SAVED_DATA_FILE_PATH = os.getenv('SAVED_DATA_FILE_PATH', FILE_PATH)
```

Comment:
please refrain from using environment variables. having a default and an option to override from the cli is good customer experience. env var doesn't work well for me, and I can only assume it won't work well for a lot of other folks.

Reply:
added `--saved-data-file-path` argument instead of env var